src/lwc/utilTemplateBuilder/utilTemplateBuilder.js

```diff
@@ -44,15 +44,34 @@ import PAYMENT_METHOD_INFO from '@salesforce/schema/DataImport__c.Payment_Method
 import DI_ACCOUNT1_IMPORTED_INFO from '@salesforce/schema/DataImport__c.Account1Imported__c';
 import DI_CONTACT1_IMPORTED_INFO from '@salesforce/schema/DataImport__c.Contact1Imported__c';
 import DI_DONATION_DONOR_INFO from '@salesforce/schema/DataImport__c.Donation_Donor__c';
+// Additional schema needed for donation donor validation
+import DI_ACCOUNT1_NAME_INFO from '@salesforce/schema/DataImport__c.Account1_Name__c';
+import DI_CONTACT1_LAST_NAME_INFO from '@salesforce/schema/DataImport__c.Contact1_Lastname__c';
 
 import CONTACT_INFO from '@salesforce/schema/Contact';
 import ACCOUNT_INFO from '@salesforce/schema/Account';
 import commonError from '@salesforce/label/c.commonError';
 import commonUnknownError from '@salesforce/label/c.commonUnknownError';
 
+// relevant Donation_Donor picklist values
```

Comment:
I don't think these constants are used anywhere else anymore, we can consolidate and have const DONATION_DONOR = {
    isAccount1: 'Account1',
    isContact1: 'Contact1'
};

Reply:
They are being used on Template Builder - i Didn't want to modify that portion of the code as well. Should I?